app/models/case_contact.rb

```diff
@@ -171,11 +193,21 @@ def reimbursement_only_when_miles_driven
     errors.add(:base, "Must enter miles driven to receive driving reimbursement.")
   end
 
+  def volunteer_address_when_reimbursement_wanted
+    return if !want_driving_reimbursement || !volunteer_address&.empty?
+
+    errors.add(:base, "Must enter a valid mailing address for the reimbursement.")
```

Comment:
```suggestion    
    if want_driving_reimbursement && volunteer_address.empty?
      errors.add(:base, "Must enter a valid mailing address for the reimbursement.")
    end
```

for readability

Reply:
Yeah, looking at it now, it's pretty gnarly.